assets/common/src/register.py

```diff
@@ -211,7 +212,9 @@ def main():
     }
     json_object = json.dumps(model_info, indent=4)
 
-    with open(registration_details, "w") as outfile:
+    registration_file =  os.path.join(registration_details, "model_registration_details.json")
```

Comment:
Please declare `model_registration_details.json` in constants.
nit: For `Path` variable use `/` op for joining over `os.path.join`